LICENSE

```diff
@@ -1,3 +1,13 @@
+Some files are licensed under GPLv3.  These files have the text
+
+    SPDX-License-Identifier: GPLv3
+
+in the file header.
+
+All other files, unless noted, are licensed under the MIT license.
+
+= GPLv3+ license text
```

Comment:
The DebOps copyright says `GPL-3.0-only`. You can't change it to GPLv3 and later, IMO. So there should not be ", or (at your option) any later version".

Reply:
Are the files with the text `SPDX-License-Identifier: GPLv3` inherited from DebOps? I actually don't even see GPLv3 listed at https://spdx.org/licenses/, where did it come from?

Reply:
I put `SPDX-License-Identifier: GPLv3` to these files, which contain some pieces from debops, but all are modified.
```
./roles/rsyslog/templates/rules.conf.j2
./roles/rsyslog/tasks/main.yml
./roles/rsyslog/tasks/deploy.yml
./roles/rsyslog/vars/inputs/remote/main.yml
./roles/rsyslog/vars/inputs/files/main.yml
./roles/rsyslog/vars/inputs/ovirt/main.yml
./roles/rsyslog/vars/inputs/viaq/main.yml
./roles/rsyslog/vars/inputs/viaq-k8s/main.yml
./roles/rsyslog/vars/inputs/basics/main.yml
./roles/rsyslog/vars/outputs/forwards/main.yml
./roles/rsyslog/vars/outputs/remote_files/main.yml
./roles/rsyslog/vars/outputs/files/main.yml
./roles/rsyslog/vars/outputs/elasticsearch/main.yml
./roles/rsyslog/vars/main.yml
```
In terms of `SPDX-License-Identifier: GPLv3`, Is the identifier `GPLv3` invalid? Should it be `GPL-3.0-only`?

Reply:
> In terms of `SPDX-License-Identifier: GPLv3`, Is the identifier `GPLv3` invalid?

I am not very familiar with SPDX, so I am not sure. Why did you considered it valid when you added it, did you have a reference for it?